asb-ballerina/Module.md

```diff
@@ -1,85 +1,116 @@
 
         2. [Get the connection string](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#get-the-connection-string)
 
-        3. [Create a queue in the Azure portal & get Entity Path](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#create-a-queue-in-the-azure-portal). It is in the format ‘queueName’.
+        3. [Create a queue in the Azure portal & get Entity Path](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#create-a-queue-in-the-azure-portal)
+           . It is in the format ‘queueName’.
 
     * For Service Bus Topics and Subscriptions
 
         1. [Create a namespace in the Azure portal](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#create-a-namespace-in-the-azure-portal)
 
         2. [Get the connection string](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#get-the-connection-string)
 
-        3. [Create a topic in the Azure portal & get Entity Path](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-topics-subscriptions-portal#create-a-topic-using-the-azure-portal). It's in the format ‘topicName‘.
+        3. [Create a topic in the Azure portal & get Entity Path](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-topics-subscriptions-portal#create-a-topic-using-the-azure-portal)
+           . It's in the format ‘topicName‘.
 
-        4. [Create a subscription in the Azure portal & get Entity Path](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-topics-subscriptions-portal#create-subscriptions-to-the-topic). It’s in the format ‘topicName/subscriptions/subscriptionName’.
+        4. [Create a subscription in the Azure portal & get Entity Path](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-topics-subscriptions-portal#create-subscriptions-to-the-topic)
+           . It’s in the format ‘topicName/subscriptions/subscriptionName’.
 
 ## Quickstart
 
 To use the Azure Service Bus connector in your Ballerina application, update the .bal file as follows:
 
 ### Step 1: Import connector
+
 Import the `ballerinax/asb` module into the Ballerina project.
+
 ```ballerina
 import ballerinax/asb as asb;
 ```
 
 ### Step 2: Create a new connector instance
-Create an `asb:AsbConnectionConfiguration` with the connection string obtained and initialize the connector with it.
+
+#### Initialize a Message Sender client
+
+This can be done providing connection string with queue or topic name.
+
 ```ballerina
-asb:AsbConnectionConfiguration config = {
-    connectionString: <CONNECTION_STRING>
-};
+asb:MessageSender messageSender = check new (connectionString, queueName);
+asb:MessageSender messageSender = check new (connectionString, TopicName);
```

Comment:
should change as
```
asb:MessageSender topicSender = check new (connectionString, TopicName);
```

Reply:
I have addressed the above change. Please check. 
/Thanks